### ISSUE TITLE
Support for Ubuntu 22.04/ROS2 Humble?

### ISSUE THREAD

**Issue by MatthijsBurgh** [TechmanRobotInc/tmr_ros2#31]
I would like to know whether there will be any support for ROS2 Humble (Ubuntu 22.04) any time soon?

**Comment by okvik** [comment]
The `tm_driver` package compiles and works with this OS/ROS combination. Haven't tried the other stuff.

**Comment by MatthijsBurgh** [comment]
Thanks @okvik. I assume that the communication with the robot is the most vulnerable to break when migrating to different OS. So I think the rest should work out of the box or with little changes.

**Comment by okvik** [comment]
@MatthijsBurgh Not really. The `tm_driver` has a, *relatively speaking*, small system / package dependency surface -- STL, POSIX stuff, basic ROS 2 APIs and messages that remained unchanged or got changed compatibly through releases, and not much more -- so it compiles and works.

The rest of the stuff has the same footprint but with additional libraries used on top, like MoveIt2 or Qt5, which might or might not give you trouble with updated packages in newer Ubuntu and whatnot...

However, you don't need anything other than `tm_driver`, `tm_msgs`, and URDF descriptions from this repo to work with these robots.

I've followed this repository for close to two years now and have soon come to conclude that the owner doesn't care about improving this code, providing tighter integration with the on-board robot system, helping the users, or even noticing, much less accepting bug fixes. Expect to be completely on your own when weird shit happens.
